contracts/paratii/UserRegistry.sol

```diff
@@ -71,9 +80,11 @@ contract UserRegistry is Ownable {
       if (_liked) {
         video.liked = true;
         video.disliked = false;
+        videoRegistry.likeVideo(_videoId);
       } else {
         video.liked = false;
         video.disliked = true;
+        videoRegistry.dislikeVideo(_videoId);
```

Comment:
we need some more logic here. If the user first liked a video, then diskliked it, the like counter in the videoregistry should be diminished.

Reply:
and vice versa mutatis mutandis.